cmd/process-agent/subcommands/check/check.go

```diff
@@ -151,31 +151,21 @@ func runCheckCmd(cliParams *cliParams) error {
 	}()
 
 	// If the sysprobe module is enabled, the process check can call out to the sysprobe for privileged stats
-	_, checks.Process.SysprobeProcessModuleEnabled = syscfg.EnabledModules[sysconfig.ProcessModule]
+	_, processModuleEnabled := syscfg.EnabledModules[sysconfig.ProcessModule]
 
-	if checks.Process.SysprobeProcessModuleEnabled {
+	if processModuleEnabled {
 		net.SetSystemProbePath(syscfg.SocketAddress)
 	}
 
-	// Connections check requires process-check to have occurred first (for process creation ts),
-	if cliParams.checkName == checks.Connections.Name() {
```

Comment:
Do we not need to have this check initialization happen anymore?

Reply:
Ah yes, I forgot to add it to the description - this was made unnecessary with https://github.com/DataDog/datadog-agent/pull/14474

Reply:
Updated!